### PR TITLE
LZ-SA-styling

### LINKED ISSUE #15: 15. As a user, I want the shopping list screen to look pretty
AC:

* The shopping list screen (the screen that lists all of the user’s purchases) looks something like the following (or equivalent, use your creativity!):

shopping-list

### PR DESCRIPTION
**Pull request by lilyhoratio**
## Description

In Lists component: 
- Updated the default colors of item sorting (inactive, next purchase sorting)
- Added fontawesome library's trash icon for the delete button
- Updated delete modal's confirmation button to pop-out to user
- Centered the "Loading" text

In Item component, updated item modal modal to look pretty.

In Search component, updated search input font color to be dark grey. 

## Related Issue

Closes #15 and #16 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before
Screen Shot 2020-07-26 at 12 39 58 AM
Screen Shot 2020-07-26 at 12 40 03 AM

### After

Screen Shot 2020-07-26 at 12 28 49 AM
Screen Shot 2020-07-26 at 12 28 55 AM

### CODE REVIEW

File: src/components/Item.js [hunk 1]
```diff
@@ -75,17 +75,27 @@ const Item = ({ item, handleChange, deleteItem }) => {
         />
       </label>
 
-      <span className={className} onClick={handleModalChange}>
+      <span
+        style={{ cursor: 'pointer' }}
```
Comment (lilyhoratio) [inline src/components/Item.js:79]: Did this because I wasn't sure how to best updated the existing className that was based on a variable.

Comment (segdeha) [reply src/components/Item.js:79]: I guess you’d have to add `cursor: pointer;` to all 4 of the classes where they’re declared? Or you could create another class for things that are `clickable` maybe?

File: src/components/List.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ const List = ({ firestore }) => {
           };
           const sortedList = sortData(data);
           return isLoading ? (
-            <p>loading...</p>
+            <p className="text-center ">Loading your shopping list...</p>
```
Comment (segdeha) [inline src/components/List.js:124]: Little details like this are what add polish to your app. Nice attention to detail!

File: src/components/List.js [hunk 3]
```diff
@@ -178,9 +178,13 @@ const List = ({ firestore }) => {
       />
       {showModal && (
         <Modal setDisplay={setModalDisplay}>
-          <h1>Warning</h1>
           <p>Are you sure you want to delete this item?</p>
-          <button onClick={() => deleteItem()}>Yes</button>
+          <button
+            className="text-primary hover:underline"
```
Comment (segdeha) [inline src/components/List.js:183]: Is `hover:underline` a valid class name? 🤔

File: src/styles/List.css [hunk 4]
```diff
@@ -11,6 +11,8 @@
 .ul-items {
   padding-left: 0rem;
   list-style: none;
+  height: 26rem;
```
Comment (segdeha) [inline src/styles/List.css:14]: I wonder if `vh` (i.e. something like `50vh` which would mean 50% of the viewport height) might be a better unit to use here. `26rem` might mean your nav is off the screen on smaller screen sizes.

File: src/components/List.js (hunk 3 above)
Comment (abidishajia) [reply src/components/List.js:183]: @segdeha Yes, [tailwind](https://tailwindcss.com/docs/text-decoration/#hover) provides us with it.

File: src/styles/List.css (hunk 4 above)
Comment (abidishajia) [reply src/styles/List.css:14]: I actually tried doing that, but then `overflow: scroll` wasn't working.

**Pull request merged**